main/webapp/modules/core/about.html

```diff
@@ -62,30 +62,32 @@
     </div>
 
     <div id="body-info">
-      <h1>About OpenRefine</h1>
+      <h1 id="about-openrefine">About OpenRefine</h1>
       <!-- The following two IDs must match those in index.vt/index.js -->
       <h2 id="openrefine-version"></h2>
       <h3 id="java-runtime-version"></h3>
       
-      <p>
+      <p id="definition">
         OpenRefine is a power tool for working with messy data. Use it to improve data consistency,
```

Comment:
Text content that is covered by i18n should not be in the HTML template anymore (otherwise people will be tempted to make edits to the HTML file directly but that will not have any effect since the text will be overridden by i18n).